amplify/backend/api/colonycdapp/schema.graphql

```diff
@@ -297,14 +297,21 @@ type User @model {
 }
 
 type Domain @model {
-  id: ID! @index(name: "byNativeId", sortKeyFields: ["nativeId"]) # colonyAddress_nativeId
+  id: ID! # We have to self-manage this and keep it the following format: colonyAddress_nativeId
+  colonyId: ID! @index(name: "byColony")
+  colony: Colony! @belongsTo(fields: ["colonyId"])
   nativeId: Int!
   nativeFundingPotId: Int!
   nativeSkillId: Int!
-  name: String
-  description: String
-  color: DomainColor
-  parent: Domain @hasOne
+  isRoot: Boolean!
```

Comment:
I thought this is an easier solution since we only support one level nesting (both in Dapp and on the contract level). Another thing that pushed me towards that change was that I couldn't get the parentId from the `DomainAdded` event or by calling `getDomain` on colony client - something to look at when we want to start supporting more complex nesting

Reply:
Sure, I'm good with that.

Just keep in mind that at some point we'll need to deal with multiple nesting levels